services/settings-service/src/test/java/com/amazon/aws/partners/saasfactory/saasboost/SettingsServiceTest.java

```diff
@@ -254,6 +254,13 @@ public void testToParameterStore() {
     public void testFromParameterStore() {
 
     }
+
+    @Test
+    public void testToSettings() {
+        AppConfig appConfig = AppConfig.builder().build();
+        List<Setting> settings = SettingsServiceDAL.toSettings(appConfig);
+        System.out.println(Utils.toJson(settings));
+    }
```

Comment:
This doesn't look to actually verify the result of the toSettings() function. Why not add keys and values of various types to the AppConfig, then verify they exist in the Setting list returned by SettingsServiceDAL?